### DISCUSSION TITLE
System.InvalidOperationException: 'No service for type 'Microsoft.AspNetCore.Hosting.Server.IServer' has been registered.'

### DISCUSSION THREAD

**Discussion by Smilefounder** [Dotnet-Boxed/Templates#176] [Q&A]
﻿**Which project template is the issue about?**
ASP.Net Core MVC
...

**Which version of the project template is the issue about?**
6.0.0.226
...

**Which version of Visual Studio or the dotnet SDK were you using?**
VS 2017 & dotnet 1.0.4
...

**Comment by RehanSaeed** [comment]
This occurs because you have no web server registered in Program.cs. You have to have at least Kestrel or WebListener.

**Comment by Smilefounder** [comment]
Thanks, let me try again and let you know the result.

**Comment by RehanSaeed** [comment]
Cool, feel free to reopen this issue.
